### ISSUE TITLE
test example

### ISSUE THREAD

**Issue by victhevenot** [learn-co-curriculum/reading-ios-using-tests#7]
would be great to see a fully written test example to follow along with the description of the test

**Comment by markedwardmurray** [comment]
There are fully written tests in this section. Do you mean somewhere else in the reading?

https://github.com/learn-co-curriculum/reading-ios-using-tests#identifying-the-test-objects

**Comment by vicfriedman** [comment]
the "Implementation Layout" section is really hard to follow without
examples of where those things are used in tests. I basically had to skip
that section because I had no frame of reference for what you were talking
about or how those things are used in the tests. Also, references to
@implementation
and @end were super confusing because I have no idea what those do in the
FISAppDelegate.m anyway.

Hope that's more clear!

**Comment by markedwardmurray** [comment]
thanks for the extra detail! Hm, it might make sense to switch the order of Using Tests with Creating Methods.
